Firmware/xyzcal.cpp

```diff
@@ -18,6 +18,8 @@
 
 #define _PINDA ((READ(Z_MIN_PIN) != Z_MIN_ENDSTOP_INVERTING)?1:0)
 
+static const char endl[2] PROGMEM = "\n";
```

Comment:
I don't think array length needs to be stated here, it can be deduced from initialization string.

Reply:
could be, you are right